assets/scss/main.scss

```diff
@@ -400,14 +427,15 @@ body {
             }
 
             @media #{$info-tablet} {
-                width: $chapter-selector-width;
+                width: calc(#{$chapter-selector-width} - 18px);
```

Comment:
the -18px seems to cause a styling issue.

![screen shot 2017-04-20 at 09 20 22](https://cloud.githubusercontent.com/assets/1399376/25232871/dac1b5c2-25aa-11e7-95de-3fbc3f80a8cb.png)